shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3533,16 +3535,17 @@ v8::Local<v8::Promise> WebContents::CapturePage(gin::Arguments* args) {
   const gfx::Size view_size =
       rect.IsEmpty() ? view->GetViewBounds().size() : rect.size();
 
-  // By default, the requested bitmap size is the view size in screen
-  // coordinates.  However, if there's more pixel detail available on the
-  // current system, increase the requested bitmap size to capture it all.
-  gfx::Size bitmap_size = view_size;
-  const gfx::NativeView native_view = view->GetNativeView();
-  const float scale = display::Screen::GetScreen()
-                          ->GetDisplayNearestView(native_view)
-                          .device_scale_factor();
-  if (scale > 1.0f)
-    bitmap_size = gfx::ScaleToCeiledSize(view_size, scale);
+  if (bitmap_size.IsEmpty()) {
+    // By default, the requested bitmap size is the view size in screen
```

Comment:
This should also perhaps be specified in the documentation - this might not be assumed behavior by users. Beyond that - is this something we want to do by default? Or is this something that we might want to delegate to users as an option when passing `width` and `height`?

Reply:
I've made changes in documentation. 

>Beyond that - is this something we want to do by default?
Not sure, I've made changes in that way to not change or affect existing apps which rely on this API. Currently, Electron does it by default.

> Or is this something that we might want to delegate to users as an option when passing width and height
For the sake of practicality I do not want to add another option to this small PR to control enablement / exposure of scaling feature. However, I am happy to raise a separate PR to introduce this option once the current PR reaches `main` :)

Reply:
Fair enough! I don't feel strongly about it really :)